.github/workflows/ci.yml

```diff
@@ -31,10 +31,13 @@ jobs:
       - uses: actions/checkout@v4
       - uses: coursier/cache-action@v6
       - uses: VirtusLab/scala-cli-setup@v1
+      - uses: actions/setup-java@v3
         with:
-          jvm: 21
+          distribution: 'zulu'
+          java-version: 21
```

Comment:
is the extra `setup-java` action necessary? it looks like previously the `jvm: 21` would handle that, without an extra action

Reply:
That is correct. I removed the java setup step.

Reply:
Removed quotes for the sbt test command.